.travis.yml

```diff
@@ -10,7 +10,18 @@ install:
 env:
   - FC=gfortran
 
+before_script:
+  - git clone https://github.com/mfem/mfem.git
+  - cd mfem
+  - make config
+  - make test -j 4
```

Comment:
To speed things up a bit, you an replace `make config; make test -j 4` with `make serial`.